library/src/main/java/com/mikepenz/fastadapter/commons/utils/RecyclerViewCacheUtil.java

```diff
@@ -35,28 +34,28 @@ public RecyclerViewCacheUtil withCacheSize(int cacheSize) {
     public void apply(RecyclerView recyclerView, Iterable<Item> items) {
         if (items != null) {
             //we pre-create the views for our cache
-            HashMap<Integer, Stack<RecyclerView.ViewHolder>> cache = new HashMap<>();
+            SparseArray<Stack<RecyclerView.ViewHolder>> cache = new SparseArray<>();
```

Comment:
"SparseArrays map integers to Objects. Unlike a normal array of Objects, there can be gaps in the indices. It is intended to be more memory efficient than using a HashMap to map Integers to Objects, both because it avoids auto-boxing keys and its data structure doesn't rely on an extra entry object for each mapping." https://developer.android.com/reference/android/util/SparseArray.html